lib/commands/update.js

```diff
@@ -92,6 +93,9 @@ class UpdateCommand extends Command {
         }, {
             title: 'Linking latest Ghost and recording versions',
             task: this.link
+        }, {
+            title: 'Installing default themes',
```

Comment:
```suggestion
            title: 'Linking built-in themes',
```

not a critical issue, but it would align with the install task 😁

Reply:
Fixed in https://github.com/TryGhost/Ghost-CLI/commit/70e3be33efbadc072f54b216c5b0ef31f5362926, thanks!